src/Components/Home/FeatureCollection.jsx

```diff
@@ -7,10 +7,12 @@ const FeatureCollection = ({ ...props }) => {
         <div className="landing-product">
             <div className="product-preview">
                 <div>
-                    <a href={"/products?category=" + collection.categoryName}><img src={collection.products[0].imagesUrl[0]} alt="product" /></a>
+                    <a href={"/products?category=" + collection.category.name + "&feature=true"}>
```

Comment:
Never use names of an object as a parameter that is sent between pages.
Names can be lengthy and, because they are strings, can contain invalid characters.
To communicate between pages, as in, click on a product, and open the corresponding product page, we pass object IDs.
This should be true for any type of object related navigation. 
If an object is passed, it needs to be passed using its ID.
So object (Name: Blue Jacket, Id: 12 ) - > `/products?id=12` instead of `/products?name=Blue%s%Jacked`

Reply:
I forgot to mention, the true representative property of an object (product, category etc) is the `ID`, so it would make sense to use that instead of anything else.